pos_accounting/src/pool/delta/delta_amount_collection.rs

```diff
@@ -126,3 +98,188 @@ impl<K: Ord> Default for DeltaAmountCollection<K> {
         }
     }
 }
+
+#[cfg(test)]
+pub mod test {
+    use super::*;
+
+    #[test]
+    fn test_add_empty() {
+        let mut collection = DeltaAmountCollection {
+            data: BTreeMap::new(),
+        };
+        collection.merge_delta_amount_element(1, SignedAmount::from_atoms(1)).unwrap();
```

Comment:
Key 1 and value 1 isn't really the clearest... maybe use some arbitrary key to make the test easier to read on the eye and distinguish between numbers that have arithmetic meaning and those that don't... like 12345? Using &str or Strings is also another good way to make that distinction.